src/diffusers/models/transformers/transformer_sd3.py

```diff
@@ -319,6 +320,11 @@ def custom_forward(*inputs):
                     hidden_states=hidden_states, encoder_hidden_states=encoder_hidden_states, temb=temb
                 )
 
+            # controlnet
+            if controlnet_block_samples is not None and not block.context_pre_only:
```

Comment:
why do we need this `interval_control` variable? 
from the implementation of sd3 controlnet, it seems `controlnet_block_samples` is fixed to have same length as `transformer_block`s